Can't build project. Path issue: build-aux

**Describe the bug**

Following the instructions in `CONTRIBUTING.md` causes the build to fail.  There is a path issue with `build-aux/ego/mkzip.sh`. `build-aux/ego/mkzip.sh` exists, but the build script is trying to invoke it after `ninja: Entering directory '_build'`, so its location is `../build-aux/ego/mkzip.sh`

**Steps To Reproduce:**

```sh
❯ git clone git@github.com:jhilliker/gnome-shell-extension-gsconnect.git  # fresh fork
Cloning into 'gnome-shell-extension-gsconnect'...
remote: Enumerating objects: 27, done.
remote: Counting objects: 100% (27/27), done.
remote: Compressing objects: 100% (27/27), done.
remote: Total 26705 (delta 10), reused 2 (delta 0), pack-reused 26678
Receiving objects: 100% (26705/26705), 26.18 MiB | 6.21 MiB/s, done.
Resolving deltas: 100% (20883/20883), done.

~/dev took 8s
❯ cd gnome-shell-extension-gsconnect/

…/gnome-shell-extension-gsconnect on  master (ee960fc) is 📦 v>=0.46.0
❯ meson _build
The Meson build system
Version: 0.57.1
Source dir: /home/j/dev/gnome-shell-extension-gsconnect
Build dir: /home/j/dev/gnome-shell-extension-gsconnect/_build
Build type: native build
Project name: gsconnect
Project version: 44
C compiler for the host machine: cc (gcc 10.2.0 "cc (GCC) 10.2.0")
C linker for the host machine: cc ld.bfd 2.36.1
Host machine cpu family: x86_64
Host machine cpu: x86_64
Program env found: YES (/usr/bin/env)
Program eslint found: NO
Program appstreamcli found: YES (/usr/bin/appstreamcli)
Configuring org.gnome.shell.extensions.gsconnect.json.google using configuration
Configuring org.gnome.shell.extensions.gsconnect.json.mozilla using configuration
Configuring metadata.json using configuration
Configuring config.js using configuration
Configuring org.gnome.Shell.Extensions.GSConnect.desktop using configuration
Configuring org.gnome.Shell.Extensions.GSConnect.Preferences.desktop using configuration
Program desktop-file-validate found: YES (/usr/bin/desktop-file-validate)
Found pkg-config: /usr/bin/pkg-config (1.7.3)
Run-time dependency dbus-1 found: YES 1.12.20
Configuring org.gnome.Shell.Extensions.GSConnect.service using configuration
Found pkg-config: /usr/bin/pkg-config (1.7.3)
Program glib-compile-resources found: YES (/usr/bin/glib-compile-resources)
Configuring config.js using configuration
Program minijasmine found: YES (/home/j/dev/gnome-shell-extension-gsconnect/installed-tests/minijasmine)
Configuring testDevice.test using configuration
Configuring testManager.test using configuration
Configuring testPacket.test using configuration
Configuring testPlugin.test using configuration
Configuring testLanBackend.test using configuration
Configuring testClipboardComponent.test using configuration
Configuring testMprisComponent.test using configuration
Configuring testBatteryPlugin.test using configuration
Configuring testClipboardPlugin.test using configuration
Configuring testContactsPlugin.test using configuration
Configuring testFindmyphonePlugin.test using configuration
Configuring testMousepadPlugin.test using configuration
Configuring testMprisPlugin.test using configuration
Configuring testNotificationPlugin.test using configuration
Configuring testPhotoPlugin.test using configuration
Configuring testPingPlugin.test using configuration
Configuring testPresenterPlugin.test using configuration
Configuring testRuncommandPlugin.test using configuration
Configuring testSftpPlugin.test using configuration
Configuring testSharePlugin.test using configuration
Configuring testSmsPlugin.test using configuration
Configuring testSystemvolumePlugin.test using configuration
Configuring testTelephonyPlugin.test using configuration
Build targets in project: 8

Found ninja-1.10.2 at /usr/bin/ninja

…/gnome-shell-extension-gsconnect on  master (ee960fc) is 📦 v>=0.46.0
❯ ninja -C _build install-zip
ninja: Entering directory '_build'
[0/1] Running external command install-zip (wrapped by meson to set env)
/usr/bin/env: ‘build-aux/ego/mkzip.sh’: No such file or directory
while executing ['/usr/bin/env', 'UUID=gsconnect@andyholmes.github.io', 'DATADIR=/usr/local/share', 'LOCALEDIR=/usr/local/share/locale', 'GSCHEMADIR=/usr/local/share/glib-2.0/schemas', 'INSTALL=true', 'build-aux/ego/mkzip.sh']
FAILED: meson-install-zip
/usr/bin/meson --internal exe --unpickle /home/j/dev/gnome-shell-extension-gsconnect/_build/meson-private/meson_exe_env_c9f28669f818e86bc2a861c74d52056c2f041d84.dat
ninja: build stopped: subcommand failed.

…/gnome-shell-extension-gsconnect on  master (ee960fc) is 📦 v>=0.46.0
1❯ ninja -C _build make-zip
ninja: Entering directory '_build'
[0/1] Running external command make-zip (wrapped by meson to set env)
/usr/bin/env: ‘build-aux/ego/mkzip.sh’: No such file or directory
while executing ['/usr/bin/env', 'UUID=gsconnect@andyholmes.github.io', 'DATADIR=/usr/local/share', 'LOCALEDIR=/usr/local/share/locale', 'GSCHEMADIR=/usr/local/share/glib-2.0/schemas', 'build-aux/ego/mkzip.sh']
FAILED: meson-make-zip
/usr/bin/meson --internal exe --unpickle /home/j/dev/gnome-shell-extension-gsconnect/_build/meson-private/meson_exe_env_dc0e0dee75855bc4d36c4791e770e94e4bce672d.dat
ninja: build stopped: subcommand failed.
```

**Expected behavior**

The build should succeed.

**System Details (please complete the following information):**

 - **GSConnect version:** v44 github master (ee960fc)
   - **Installed from:**  github master (ee960fc)
 - **GNOME/Shell version:** n/a
 - **Distro/Release:** Arch

**Additional Notes:**

PR to follow...

--- a/meson.build
+++ b/meson.build
@@ -66,7 +66,7 @@ run_target(
     'DATADIR=' + datadir,
     'LOCALEDIR=' + localedir,
     'GSCHEMADIR=' + gschemadir,
-    'build-aux/ego/mkzip.sh'
+    join_paths(meson.source_root(), 'build-aux', 'ego', 'mkzip.sh')
   ]
 )
 
@@ -79,7 +79,7 @@ run_target(
     'LOCALEDIR=' + localedir,
     'GSCHEMADIR=' + gschemadir,
     'INSTALL=true',
-    'build-aux/ego/mkzip.sh'
+    join_paths(meson.source_root(), 'build-aux', 'ego', 'mkzip.sh')
   ]
 )
 

